test/os/unit/unitmanager.test.js

```diff
@@ -78,6 +78,9 @@ describe('os.unit.UnitManager', function() {
     expect(um.formatToBestFit('distance', 1000, 'm', um.getBaseSystem(), 0)).toBe('1 km');
     expect(um.formatToBestFit('distance', 999, 'm', um.getBaseSystem(), 0)).toBe('999 m');
 
+    // Test zero to default
+    expect(um.formatToBestFit('distance', .0, 'm', um.getBaseSystem(), 2)).toBe('0.00 m');
```

Comment:
Out of curiosity, why `.0` instead of `0`?

Reply:
Its my C# / C++ background - conceptually `double` instead of `int`.

Reply:
Ohh, roger that. Carry on.